controllers/bootstrap/init.go

```diff
@@ -1505,7 +1552,7 @@ func (b *Bootstrap) updateApprovalMode() error {
 		return err
 	}
 
-	if err = b.UpdateOpApproval(constant.IBMCSPackage); err != nil {
+	if err = b.UpdateCsOpApproval(); err != nil {
 		klog.Errorf("Failed to update %s subscription: %v", constant.IBMCSPackage, err)
 		return err
 	}
```

Comment:
Could we update Cs-operator after we update the ODLM and NSS operator?
Because the code logic will restart the pod of cs-operator before running into updating ODLM and NSS operator